test/integration/sanitation-test.coffee

```diff
@@ -48,29 +48,30 @@ describe('Sanitation of Reported Data', ->
       }
     })
 
-  # helper for preparing server
-  createServer = (response) ->
-    app = express()
+  # helper for preparing the server under test
+  createServerFromResponse = (response) ->
+    app = createServer()
     app.put('/resource', (req, res) -> res.json(response))
     return app
 
 
   describe('Sanitation of the Entire Request Body', ->
-    results = undefined
+    dreddRuntimeInfo = undefined
 
     beforeEach((done) ->
-      dredd = createDredd('entire-request-body')
-      app = createServer({name: 123}) # 'name' should be string → failing test
+      dredd = createDreddFromFixture('entire-request-body')
+      app = createServerFromResponse({name: 123}) # 'name' should be string → failing test
 
-      runDreddWithServer(dredd, app, (args...) ->
-        [err, results] = args
-        done(err)
+      runDreddWithServer(dredd, app, (err, runtimeInfo) ->
+        return done(err) if err
```

Comment:
It's not always possible. There are two different situations:

1. The callback receives `(err, object)` and I need the `object`:

```coffee
object = undefined
fn((err, obj) ->
  object = obj
  done(err)
)
```

2. The callback receives `(err, object)` and I need a property of the `object`:

```coffee
property = undefined
fn((err, obj) ->
  return done(err) if err
  property = obj.property
  done()
)
```

If I skipped the `return done(err) if err` in the second case, the line `property = obj.property` would blow up 💥 (accessing a `property` on `null` or `undefined`).